modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties

```diff
@@ -120,7 +120,7 @@
     # Liferay CE release and "7.1.10" for the latest DXP release.
     #
 #if (${liferayVersion.startsWith("7.0")})
-    #liferay.workspace.target.platform.version=7.0-GA7
+    #liferay.workspace.target.platform.version=7.0.6.1
```

Comment:
Lets do `7.0.6` and also lets update the documentation in `gradle.properties`

Reply:
@gamerson two questions here:
1. According to Nexus, the latest version of the BOM artifact is `7.0.6.1`. I was of the understanding that this updated version fixed an issue with `7.0.6`. Am I misunderstanding how this works?
2. Do you want the documentation for this property to include both 7.0 and 7.1 **latest** CE and DXP released BOM versions? Is that what you mean by updating the documentation in `gradle.properties`?

Reply:
We fixed the 7.0.6 artifact on our servers, so it should be safe to use 7.0.6.

And yes we need to update the embedded documentation to just use the simpler version numbers.